docs/sphinx/examples/python/tutorials/Shors.ipynb

```diff
@@ -21,15 +21,35 @@
   },
   {
    "cell_type": "markdown",
-   "id": "0f979d1e-57a1-48ca-a7ae-efb3750c2b2f",
+   "id": "148568d8",
    "metadata": {},
    "source": [
     "First let's install some libraries."
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
+   "execution_count": 155,
+   "id": "486fb3d4",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Requirement already satisfied: contfrac in /usr/local/lib/python3.10/dist-packages (1.0.0)\n",
+      "\u001b[33mWARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv\u001b[0m\u001b[33m\n",
+      "\u001b[0m"
```

Comment:
I would recommend clearing this output cell in order to clean up the web page that will be generated from this notebook. (No need for regular web page viewers to see these warnings about broken things 😄 ).